ambry-tools/src/main/java/com.github.ambry/store/ConsistencyCheckerTool.java

```diff
@@ -57,38 +64,105 @@
    modified times of the files unchanged. In linux use "cp -p" to maintain the file attributes.
    */
 
+  /**
+   * Config for the ConsistencyCheckerTool.
+   */
+  private static class ConsistencyCheckerToolConfig {
+
+    /**
+     * The path where the input is.
+     */
+    @Config("path.of.input")
```

Comment:
can you fix documentation in lines 48, 49. partition root dir is not valid anymore right.

Reply:
Done